src/trpc/router/vacancyRouter.ts

```diff
@@ -69,9 +70,13 @@ export const vacancyRouter = router({
     }),
 
     archive: protectedProcedure.input(z.string()).mutation(async ({ input, ctx }) => {
+        const vacancy = await vacancyMethods.getById(input);
         accessCheckAnyOf(
             checkRoleForAccess(ctx.session.user.role, 'editFullGroupTree'),
-            checkRoleForAccess(ctx.session.user.role, 'editAdministratedGroupTree'),
+            accessCheckAllOf(
```

Comment:
maybe this group edit check can be move to a separate function, since it's called in so many places